docs/yaml.md

```diff
@@ -58,6 +58,21 @@ build:
   python_version: "3.8.1"
 ```
 
+### `run`
+
+A list of commands to run to run in the environment to set it up. They are run last, after your system packages and Python packages have been installed. If you've used Docker, it's just like a `RUN` instruction in your `Dockerfile`.
+
+Note that your code is _not_ available to commands. This is so we can build your image efficiently when running locally.
```

Comment:
I was confused by the phrasing "to commands". Perhaps "to commands in the `run` clause"

Reply:
Clumsy "note that" too, reading again.